src/OpenFeature.DependencyInjection/MultiTarget/CallerArgumentExpressionAttribute.cs

```diff
@@ -0,0 +1,23 @@
+// @formatter:off
+// ReSharper disable All
+#if NETCOREAPP3_0_OR_GREATER
+// https://github.com/dotnet/runtime/issues/96197
+[assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Runtime.CompilerServices.CallerArgumentExpressionAttribute))]
+#else
+#pragma warning disable
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Runtime.CompilerServices;
+
+[AttributeUsage(AttributeTargets.Parameter, AllowMultiple = false, Inherited = false)]
+internal sealed class CallerArgumentExpressionAttribute : Attribute
```

Comment:
Please consider using [PolySharp](https://www.nuget.org/packages/PolySharp/#readme-body-tab). It is smart and customizable source generator which will detect what exact feature need to be poly-filled and generate the code for that.

Reply:
Thanks for the suggestion! I’ll definitely take a closer look and see how it can fit into our current approach.

Reply:
@arttonoyan / @wwalendz-relativity, thanks for bringing this to our attention. I need someone else to confirm if bringing an extra library to use these polyfils is okay. 

@toddbaert or @beeme1mr ?

Reply:
Conceptually I think I am fine with a library like this. If it can be used as a generator, and doesn't have a runtime dependency, then that seems reasonable.

The important thing is to ensure that it doesn't cause friction with someone who wants to use this library.

I would be skeptical of bundling addition of that dependency with a feature implementation though.